app/assets/javascripts/index.js

```diff
@@ -238,11 +238,37 @@ $(document).ready(function () {
     L.marker(center, {icon: OSM.getUserIcon()}).addTo(map);
   });
 
+  var isMobile = {
+    Windows: function() {
+        return /IEMobile/i.test(navigator.userAgent);
+    },
+    Android: function() {
+        return /Android/i.test(navigator.userAgent);
+    },
+    BlackBerry: function() {
+        return /BlackBerry/i.test(navigator.userAgent);
+    },
+    iOS: function() {
+        return /iPhone|iPad|iPod/i.test(navigator.userAgent);
+    },
+    any: function() {
+        return (isMobile.Android() || isMobile.BlackBerry() || isMobile.iOS() || isMobile.Windows());
+    }
+  };
+
+  function url() {
```

Comment:
This is far too vague as a name for this function at this scope - it should either be more explanatory or the function should be pushed down inside `remoteEditHandler` so that the meaning is implicit.

That said I'm not really sure we it needs to be a function and can't just be done inline in `remoteEditHandler` without having to repeat the android test...

Reply:
No using leaflet internals for non-leaflet things is a terrible idea.